### PR TITLE
fix(Maps/MeshExtractor): build

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Restore some old code
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3329

## Tests Performed:

- builds
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/tools/mesh_extractor/ContinentBuilder.cpp [hunk 1]
```diff
@@ -191,7 +191,7 @@ void ContinentBuilder::Build()
                     if ((*_th)->Free)
                     {
                         (*_th)->SetData(itr->X, itr->Y, MapId, Continent);
-                        (*_th)->activate();
+                        // (*_th)->activate(); // Not defined. Leftover from ACE remove?
```
Comment (Kitzunu) [inline src/tools/mesh_extractor/ContinentBuilder.cpp:194]: same as above

**Comment by IntelligentQuantum** [comment]
I will delete the MeshExtractor folder in future updates

**Comment by Kitzunu** [comment]
But now, it is broken

**Comment by IntelligentQuantum** [comment]
Well, uncheck it in cmake so that it does not make error
For the map, vmaps, mmaps, just check the TOOLS, MESH is useless tool

**Pull request closed without merging**